client/web/admin/src/components/CRolePicker.vue

```diff
@@ -47,8 +47,6 @@
 <script>
 import { debounce } from 'lodash'
 import { VueSelect } from 'vue-select'
-import { components } from '@cortezaproject/corteza-vue/'
-const { CInputConfirm } = components
 
 function roleSorter (a, b) {
```

Comment:
Can you also remove the no-prompt from c-input confirm in this file
Same for CRoleEditorMembers c-input-confirm